fingerprintauthhelper/src/main/kotlin/com/pro100svitlo/fingerprintAuthHelper/FahManager.kt

```diff
@@ -303,11 +290,11 @@ internal class FahManager(@NonNull c: Context, l: FahListener?, keyName: String,
             }
 
             try {
-                mKeyStore!!.load(null)
+                keyStore.load(null)
```

Comment:
As the local variable and the property are referencing the same object, this would also affect the property.